test/unit/TokensTest.php

```diff
@@ -643,6 +644,162 @@ public function testObtainCachedToken()
         $this->assertEquals($cachedTokenDetails, $tokenDetails);
     }
 
+    public function testRemoteValidationCallsUseSuScope()
+    {
+        $mockClient = $this->getMock(
+            'Talis\Persona\Client\Tokens',
+            ['personaCheckTokenIsValid', 'validateTokenUsingJWT'],
+            [[
+                'userAgent' => 'unittest',
+                'persona_host' => 'localhost',
+                'persona_oauth_route' => '/oauth/tokens',
+                'cacheBackend' => $this->cacheBackend,
+            ]]
+        );
+
+        $accessToken = json_encode(
+            [
+                'access_token' => JWT::encode(
+                    [
+                        'jwtid' => time(),
+                        'exp' => time() + 100,
+                        'nbf' => time() - 1,
+                        'audience' => 'standard_user',
+						'scopeCount' => 50,
+                    ],
+                    $this->_privateKey,
+                    'RS256'
+                ),
+                'expires_in' => 100,
+                'token_type' => 'bearer',
+            ]
+        );
+
+        $expectedValidationUrl = 'localhost/oauth/tokens/'
+            . $accessToken
+            . '?scope=su,invalidScope';
+
+		$mockClient
+			->expects($this->once())
+			->method('validateTokenUsingJWT')
+			->will($this->throwException(new ScopesNotDefinedException()));
```

Comment:
This test checks for a positive response, while the test below tests for a negative response